### ISSUE TITLE
"disabled" not handled

### ISSUE THREAD

**Issue by UltraBlackLinux** [Venom1991/refind-btrfs#39]
the keyword `disabled` (see refind sample config) is not handled. Instead refind-btrfs returns `ERROR (refind_btrfs.state_management.refind_btrfs_machine/refind_btrfs_machine.py/run): Boot stanza is not matched with the partition!`

**Comment by Venom1991** [comment]
These boot stanzas (the ones in which the "disabled" keyword is present) are completely ignored. This error could be caused by a legitimate problem with a found boot stanza.

**Comment by UltraBlackLinux** [comment]
adding "disabled" causes the error. Removing it also removes the error. The boot stanza is bootable and works as expected

**Comment by Venom1991** [comment]
I don't quite understand what you're trying to say. Boot stanzas with the "disabled" keyword are ignored by both rEFInd and refind-btrfs.

**Comment by UltraBlackLinux** [comment]
I found out, what's happening: If two menu entries exist which both boot the same kernel, an error is thrown. Naturally you `disable` one entry. You are now left with a different error: `ERROR: Boot stanza is not matched with the partition!`

Tl;Dr: if two menu options with the same kernel exist, even though one is disabled, an error is thrown. One that doesn't even make much sense from a user's perspective
